src/OrchardCore.Modules/OrchardCore.Html/Migrations.cs

```diff
@@ -54,7 +54,7 @@ public int UpdateFrom2()
         }
 
         // This code can be removed in a later version.
-        public async Task<int> UpdateFrom3()
```

Comment:
How did this work before?

Reply:
Because support both with Async and without. But it is better to use the Async prefix for consistency

Reply:
Ah OK.